@ecomplus/storefront-renderer/functions/index.js

```diff
@@ -49,6 +68,10 @@ exports.ssr = (req, res, getCacheControl) => {
   const fallback = () => {
     if (url.slice(-1) === '/') {
       redirect(url.slice(0, -1))
+    } else if (url.indexOf('/proxy-reverse/') > -1) {
+      const script = partytownProxyReverse(url)
+      setStatusAndCache(200, `public, max-age=${(isLongCache ? 120 : 30)}`)
```

Comment:
E pode forçar cache nisso? 👀 
O correto seria você respeitar os cabeçalhos de cache do backend em vez de forçar um aqui.

Reply:
A reposta do proxy reverso deveria incluir os headers (incluindo de cache), não apenas o body, originais do backend (o `url`)

Reply:
Posso receber o retorno e mandar as is para a requisição original?

Reply:
Uai não só pode como deve, proxy reverso é isso 😄